src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
```

Comment:
Calling `ev.preventDefault()` here prevents the default action of the event, which is correct if you want to stop the browser from following the link when a thumbnail is clicked. However, ensure that this behavior is intended for all clicks within the `thumbs` element.